docs/featurestore/featurestore.rst

```diff
@@ -834,16 +834,20 @@ Follow the steps described in `Connecting from Amazon SageMaker`_ for setting up
 
 The feature store library needs to be installed to connect to it. In the Databricks UI, go to *Clusters* and select your cluster. Select *Libraries* and then *Install New*. As *Library Source* choose *PyPI* and fill in *hopsworks-cloud-sdk* into the *Package* field.
 
-**Mounting a bucket for storing certificates**
+**Creating a DBFS folder to store certificates**
+
+Hopsworks relies on certificates being available in the Databricks cluster to connect to some services inside Hopsworks. To ensure that these certificates can be distributed to all nodes in a Databricks cluster, Hopsworks relies on them being stored in the Databricks file system.
+For this purpose, you should create a cert folder in DBFS. This can be done by running the following in a Databricks notebook connected to the prepared cluster::
+
+    dbutils.fs.mkdirs("dbfs:/certs/")
 
-Hopsworks relies on certificates being available in the Databricks cluster in order to connect to some services inside Hopsworks. To ensure that these certificates can be distributed to all nodes in a Databricks cluster, Hopsworks relies on mounting an S3 bucket with read/write permission using the databricks file system. Please follow Databrick's guide for setting up a mount: `Mount S3 Buckets with DBFS <https://docs.databricks.com/data/data-sources/aws/amazon-s3.html#mount-s3-buckets-with-dbfs>`_
 
 **Connecting to the Feature Store**
 
 In the Databricks notebooks connected to the prepared cluster use the following code to connect to the feature store::
 
     import hops.featurestore as fs
-    fs.connect('my_instance.us-east-2.compute.amazonaws.com', 'my_project', cert_folder='/dbfs/mnt/my_mount_name')
+    fs.connect('my_instance.us-east-2.compute.amazonaws.com', 'my_project', cert_folder='/dbfs/certs/')
```

Comment:
Is mnt optional?

Reply:
mnt is for when you mount your own bucket, but here we put in the bucket created by databricks so we don't need the mnt.

Reply:
:+1: